integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2766,38 +2762,38 @@ msgstr ""
 
 #: cms/models/pages/page.py:163
 msgid "editors"
-msgstr "Bearbeiter"
+msgstr "Redakteur*innen"
 
 #: cms/models/pages/page.py:165
 msgid "A list of users who have the permission to edit this specific page."
 msgstr ""
-"Eine Liste von Benutzern, die die Berechtigung haben, diese bestimmte Seite "
-"zu editieren."
+"Eine Liste von Konten, die die Berechtigung haben, diese bestimmte Seite zu "
+"editieren."
 
 #: cms/models/pages/page.py:167
 msgid ""
 "Only has effect if these users do not have the permission to edit pages "
 "anyway."
 msgstr ""
-"Dies betrifft nur Benutzer, die nicht ohnehin die Berechtigung haben, "
+"Dies betrifft nur Konten, die nicht ohnehin die Berechtigung haben, "
 "beliebige Seiten zu ändern."
 
 #: cms/models/pages/page.py:175
 msgid "publishers"
-msgstr "Veröffentlicher"
+msgstr "Veröffentlicht von: "
```

Comment:
```suggestion
msgstr "Redakteur*innen"
```
The problem is that `publishers` does not contain all people which actually published the page, but people who have the permission to publish the page (in other words - the editors), so the `editors` fields is a bit misleading at the moment, because it contains all `authors`... not your fault, I will open another issue for it.